app/views/trips/show.html.erb

```diff
@@ -147,11 +147,23 @@
         </div>
       </div>
     </div>
-    <h4>Recommended for Your Activities:</h4>
+    <h4>Smart Recommendations:</h4>
```

Comment:
I see. I didn't think too much about what "Smart" and "Curated" each meant, just wanted to leave some HTML there so we know they are two different things. I think "Curated Recommendations" is good but "Smart Recommendations" might be a bit vague. If I were a user I'd be guessing what the "Smart" list is about.

Also, since the checklist populates itself upon trip creation, the "Curated Recommendations" is almost going to be always empty(exceptions will be like cases where you delete an item from the checklist). Any thoughts?